man/IDateTime.Rd

```diff
@@ -85,6 +85,7 @@ year(x)
     \code{as.IDate.default}, arguments are passed to \code{as.Date}. For
     \code{as.ITime.default}, arguments are passed to \code{as.POSIXlt}.}
   \item{tz}{time zone (see \code{strptime}).}
+  \item{use_merge}{ Should the parsing be done by via merge? See Details. }
```

Comment:
It ts good the mention what data type it expects, I usually do it as the first word of argument description. You can fix that for `tz` argument also.
I also don't like `use_merge` name, maybe `use.dict`?
Instead of character we could use a numeric value to avoid hardcoded 1000 length to kick in the merge. Default to 1000, passing `Inf` would be equivalent to current FALSE. I can be easier for users to program around that switch knowing uniqueN of input.

Reply:
Feels a bit weird to me to use absolute numbers like that... in terms of knowing your data you can always pass `TRUE/FALSE` (I mainly have in mind `FALSE` to account for the case when you know you have very few repeated dates). In any case moving to this I'd change the argument name to `dict.threshold` or similar.